betting-game-react/src/App.test.js

```diff
@@ -4,6 +4,6 @@ import App from './App';
 
 test('renders learn react link', () => {
```

Comment:
I'm not a fan of these types of tests anyways - as we already discussed - keep it in if you want to :).
Please update the test name though, it is currently not correct anymore.

Reply:
Has been changed. Hope this is better :-)